app/views/decidim/action_delegator/_delegations_modal.html.erb

```diff
@@ -0,0 +1,95 @@
+                <%= button_to decidim_consultations.question_question_votes_path(question),
+                              method: :delete,
+                              remote: true,
+                              data: { disable: true },
+                              class: "button expanded success",
+                              id: "delegation_unvote_button" do %>
+                  <%= hidden_field_tag :decidim_consultations_delegation_id, delegation.id, id: "decidim_consultations_delegation_id_delete" %>
+                  <div class="vote-button-caption"
+                       data-original="<%= t("questions.vote_button.already_voted", scope: "decidim") %>"
+                       data-replace="<%= t("questions.vote_button.already_voted_hover", scope: "decidim") %>">
+                    <%= t("questions.vote_button.already_voted", scope: "decidim") %>
+                  </div>
+                <% end %>
+              <% elsif allowed_to? :vote, :question, { question: question }, [Decidim::Consultations::Permissions, Decidim::Admin::Permissions, Decidim::Permissions], delegation.granter %>
+                <% if question.multiple? %>
+                  <%= link_to decidim_consultations.question_question_multiple_votes_path(question),
+                              class: "button expanded",
+                              id: "multivote_button" do %>
+                    <div class="vote-button-caption"><%= t "questions.vote_button.vote", scope: "decidim" %></div>
+                  <% end %>
+                <% else %>
+
+                  <%# This link is slightly modified from _vote_button to add stuff needed by delegations %>
+                  <%= link_to "#", class: "button expanded delegation-vote-button", data: { "delegation-id" => delegation.id, "delegation-granter-name" => delegation.granter.name } do %>
+                    <div class="vote-button-caption"><%= t "questions.vote_button.vote", scope: "decidim" %></div>
+                  <% end %>
+
+                <% end %>
+              <% else %>
+                <%= authorized_vote_modal_button(question, remote: true, class: "button expanded light button--sc") do %>
+                  <%= t("questions.vote_button.verification_required", scope: "decidim") %>
+                <% end %>
+              <% end %>
+            <% elsif question.consultation.active? %>
+              <%= logged_button_to decidim_consultations.question_question_votes_path(question),
+                                   class: "button expanded",
+                                   data: { disable: true },
+                                   id: "vote_button" do %>
+                <div class="vote-button-caption">
+                  <%= t("questions.vote_button.vote", scope: "decidim") %>
+                </div>
+              <% end %>
+            <% end %>
+
+          </div>
+        </div>
+      <% end %>
```

Comment:
this is the bit that worries me the most. I copied it but then added stuff for delegations on top, so we diverged a bit.